game/src/starter/Main.java

```diff
@@ -0,0 +1,12 @@
+package starter;
+
+import configuration.KeyboardConfig;
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        // start the game
+        Game.loadConfig("dungeon_config.json", KeyboardConfig.class);
```

Comment:
Den Aufruf vielleicht sogar in die `Game.run()`?
Und s.o.

Reply:
Hab ich auch drüber nachgedacht, aber mir (und so wie @cagix verstanden habe ihn auch) scheint mir das hier mehr im Sinne von #582 zu sein,

Reply:
@fwatermann ich fand die idee ganz charmant, in game die konfiguration etc. zunächst von außen anzustoßen und schrittweise zu vervollständigen und dann am ende das game zu starten.

das ist im prinzip das builder-pattern: du rufst alle möglichen statischen methoden auf, und erst der letzte aufruf von "build" baut das objekt fertig - das wäre hier der "run"-aufruf.

Reply:
@fwatermann @AMatutat außerdem passt das besser zu dem ziel, dass "game" in die api wandern und nicht mehr von den nutzern verändert bzw. überschrieben werden soll. 

d.h. man baut sich in "main" mit verschiedenen builder-methoden das game zusammen und startet es dann am ende.

neben der reinen konfiguration hatten wir auch an die übergabe von objekten oder methoden gedacht, wo sich das anbietet.